CREATE-NEW-SCALER.md

```diff
@@ -8,17 +8,13 @@ In order to develop a scaler, a developer should do the following:
 2. Create the new scaler struct under the `pkg/scalers` folder.
 3. Implement the methods defined in the [scaler interface](#scaler-interface) section.
 4. Create a constructor according to [this](#constructor).
-5. Change the `getScaler` function in `pkg/scaling/scale_handler.go` by adding another switch case that matches your scaler.
+5. Change the `getScaler` function in `pkg/scaling/scale_handler.go` by adding another switch case that matches your scaler. Scalers in the switch are ordered alphabetically, please follow the same pattern.
 6. Run `make build` from the root of KEDA and your scaler is ready.
 
 If you want to deploy locally
-1. Run `export VERSION=local`
-2. Open the terminal and go to the root of the source code
-3. Run `make build`
-5. If you haven't done it yet clone the charts repository: `git  clone git@github.com:kedacore/charts.git`
-6. In the terminal, navigate to the `chart/keda` folder (the charts downloaded in step 3), and run the following command (don't forget to replace the placeholder text in the command) `helm install . --set image.keda=kedacore/keda:[tag used in step 1],image.pullPolicy=IfNotPresent`.
-
-The last step assumes that you have `helm` already installed in the cluster. In this step we install the helm chart, and we substitute the image with the image we built in step 1. Notice that we are also overriding the image PullPolice to `IfNotPresent` since this is a local cluster.
+1. Open the terminal and go to the root of the source code
+2. Run `IMAGE_REPO=johndoe make publish`, where `johndoe ` is your GitHub repo, this will create and publis images with your build of KEDA into your repo. Please refer [the guide for local deployment](Readme.md#deploying-custom-keda-locally-outside-cluster) for more details. 
```

Comment:
`is your GitHub repo`

you mean DockerHub, right?

Reply:
Good catch, thanks!